### DISCUSSION TITLE
Semantic AD only has semantic shift (i.e. P(Y) is not equal to P'(Y))?

### DISCUSSION THREAD

**Discussion by XixiLiu95** [Jingkang50/OODSurvey#12] [General]
Hi, 

I think this statement might be partially correct. In the case of semantic shift (Out-of-distribution detection), the reason that P(Y) is not equal to P'(Y)  is that the input image x is different from the training data. It means that P(x) is also not equal to P'(x). For example, the network is trained on dogs and cats.  Two input images are given, x1 denotes a dog, x2 denotes a bird. It is also expected that P(x1) > P(x2). 

Best,
Xixi

**Comment by Jingkang50** [comment]
Agree, when p(y) changes, p(x) Will be changed accordingly. We are revising the paper and will make this sentence more rigorous. Thanks!

**Comment by huytransformer** [comment]
What about this scenario: The distribution of features remains the same but new associations emerge (ie P(Y) changes)?
